### PR TITLE
[docs] Migration from EE to CSE

### PR DESCRIPTION
**Pull request by sprait**
## Description

Add instructions about migrate from EE to CSE.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: feature
summary: Add instructions on how to migrate from EE to CSE.
impact_level: low
```

### CODE REVIEW

File: docs/documentation/pages/DECKHOUSE-FAQ_RU.md [hunk 1]
```diff
@@ -965,6 +965,228 @@ kubectl -n d8-system exec -ti svc/deckhouse-leader -c deckhouse -- deckhouse-con
...
+) ) на альтернативаные варианты.
+
+    Отключить не поддерживаемые модули в CSE редакции можно следующими командами:
+
+   ```shell
+   kubectl -n d8-system exec -it deploy/deckhouse bash 
+
+   kubectl get modules.deckhouse.io | grep Enabled | grep -v -E "(admission-policy-engine|cni-flannel|common|control-plane-manager|chrony|deckhouse|ingress-nginx|kube-dns|kube-proxy|log-shipper|local-path-provisioner|loki|monitoring-kubernetes|node-manager|operator-prometheus|registrypackages|runtime-audit-engine|operator-trivy|prometheus|user-authn|user-authz|priority-class|cloud-data-crd|operator-prometheus-crd|prometheus-crd|user-authn-crd|vertical-pod-autoscaler-crd|flow-schema|prometheus-metrics-adapter|monitoring-deckhouse|monitoring-kubernetes-control-plane|gost-integrity-controller|snapshot-controller-crd)" | awk {'print "deckhouse-controller module disable ", $1'} | bash && exit
```
Comment (RomanenkoDenys) [inline docs/documentation/pages/DECKHOUSE-FAQ_RU.md:994]: - Firstly we need to get unsupported modules list to solve problems before release change.

File: docs/documentation/pages/DECKHOUSE-FAQ_RU.md [hunk 2]
```diff
@@ -965,6 +965,228 @@ kubectl -n d8-system exec -ti svc/deckhouse-leader -c deckhouse -- deckhouse-con
...
+
+    Пример:
+
+   ```console
+   # journalctl -u bashible -n 5
+   Aug 21 11:04:28 master-ee-to-cse-0 bashible.sh[53407]: Configuration is in sync, nothing to do.
+   Aug 21 11:04:28 master-ee-to-cse-0 bashible.sh[53407]: Annotate node master-ee-to-cse-0 with annotation node.deckhouse.io/configuration-checksum=9cbe6db6c91574b8b732108a654c99423733b20f04848d0b4e1e2dadb231206a
+   Aug 21 11:04:29 master-ee-to-cse-0 bashible.sh[53407]: Succesful annotate node master-ee-to-cse-0 with annotation node.deckhouse.io/configuration-checksum=9cbe6db6c91574b8b732108a654c99423733b20f04848d0b4e1e2dadb231206a
+   Aug 21 11:04:29 master-ee-to-cse-0 systemd[1]: bashible.service: Deactivated successfully.
+   ```
```
Comment (RomanenkoDenys) [inline docs/documentation/pages/DECKHOUSE-FAQ_RU.md:1051]: also we need to check all nodegroups to see that desired relicas = actual

File: docs/documentation/pages/DECKHOUSE-FAQ_RU.md [hunk 3]
```diff
@@ -965,6 +965,228 @@ kubectl -n d8-system exec -ti svc/deckhouse-leader -c deckhouse -- deckhouse-con
...
+            sandbox_image = "registry-cse.deckhouse.ru/deckhouse/cse@$CSE_SANDBOX_IMAGE"
+        EOF_TOML
+
+        sed -i 's|image: .*|image: registry-cse.deckhouse.ru/deckhouse/cse@$CSE_K8S_API_PROXY|' /var/lib/bashible/bundle_steps/051_pull_and_configure_kubernetes_api_proxy.sh
+        sed -i 's|crictl pull .*|crictl pull registry-cse.deckhouse.ru/deckhouse/cse@$CSE_K8S_API_PROXY|' /var/lib/bashible/bundle_steps/051_pull_and_configure_kubernetes_api_proxy.sh
+   EOF
+   ```
+
+    В журнале systemd-сервиса bashible на master-узле должно появиться сообщение `Configuration is in sync, nothing to do`.
+
```
Comment (RomanenkoDenys) [inline docs/documentation/pages/DECKHOUSE-FAQ_RU.md:1099]: also wait nodegroups

File: docs/documentation/pages/DECKHOUSE-FAQ_RU.md [hunk 4]
```diff
@@ -965,6 +965,228 @@ kubectl -n d8-system exec -ti svc/deckhouse-leader -c deckhouse -- deckhouse-con
...
+   ```console
+   kubectl replace -f /tmp/cse-deckhouse-registry.yaml
+   ```
+
+1. Примените CSE образ:
+
+   ```console
+   kubectl -n d8-system set image deployment/deckhouse deckhouse=registry-cse.deckhouse.ru/deckhouse/cse:v1.58.2
+   ```
```
Comment (RomanenkoDenys) [inline docs/documentation/pages/DECKHOUSE-FAQ_RU.md:1131]: in this step we should delete init container for external modules

**Review by RomanenkoDenys — COMMENTED** [review]
some comments

File: docs/documentation/pages/DECKHOUSE-FAQ_RU.md (hunk 2 above)
Comment (sprait) [reply docs/documentation/pages/DECKHOUSE-FAQ_RU.md:1054]: done, desired status doesn't work for static node groups, but `UPTODATE` show what is needed.

File: docs/documentation/pages/DECKHOUSE-FAQ_RU.md (hunk 3 above)
Comment (sprait) [reply docs/documentation/pages/DECKHOUSE-FAQ_RU.md:1139]: done.

File: docs/documentation/pages/DECKHOUSE-FAQ_RU.md (hunk 1 above)
Comment (sprait) [reply docs/documentation/pages/DECKHOUSE-FAQ_RU.md]: done, I deleted hardcode with list of unsupported modules and add point about check modules which will be disabled:
https://github.com/deckhouse/deckhouse/blob/aaaa00124487f365b106980ab5257fa5d19d1088/docs/documentation/pages/DECKHOUSE-FAQ_RU.md?plain=1#L1076-L1090

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: docs/documentation/pages/DECKHOUSE-FAQ_RU.md [hunk 5]
```diff
@@ -965,7 +965,269 @@ kubectl -n d8-system exec -ti svc/deckhouse-leader -c deckhouse -- deckhouse-con
...
+    # journalctl -u bashible -n 5
+    Aug 21 11:04:28 master-ee-to-cse-0 bashible.sh[53407]: Configuration is in sync, nothing to do.
+    Aug 21 11:04:28 master-ee-to-cse-0 bashible.sh[53407]: Annotate node master-ee-to-cse-0 with annotation node.deckhouse.io/configuration-checksum=9cbe6db6c91574b8b732108a654c99423733b20f04848d0b4e1e2dadb231206a
+    Aug 21 11:04:29 master-ee-to-cse-0 bashible.sh[53407]: Succesful annotate node master-ee-to-cse-0 with annotation node.deckhouse.io/configuration-checksum=9cbe6db6c91574b8b732108a654c99423733b20f04848d0b4e1e2dadb231206a
+    Aug 21 11:04:29 master-ee-to-cse-0 systemd[1]: bashible.service: Deactivated successfully.
+    ```
+
+5. Актуализируйте секрет доступа к registry Deckhouse, выполнив следующие команды:
+
+   ```console
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-FAQ_RU.md:1152]: @sprait can we merge two commands into one?  Like `kubectl  create... -o yaml | kubectl replace -f -`  ?

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [9581](https://github.com/deckhouse/deckhouse/pull/9581) to the branch [release-1.63](https://github.com/deckhouse/deckhouse/tree/release-1.63) successful!

**Pull request merged**